packages/flutter_tools/templates/create/ios-objc.tmpl/Podfile

```diff
@@ -1,36 +0,0 @@
-# Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
-
-if ENV['FLUTTER_FRAMEWORK_DIR'] == nil
-  abort('Please set FLUTTER_FRAMEWORK_DIR to the directory containing Flutter.framework')
-end
-
-target 'Runner' do
-  # Pods for Runner
```

Comment:
The Obj-C and Swift Podfiles weren't identical. Have we fixed the `use_frameworks!` issue?